editor/plugins/asset_library_editor_plugin.cpp

```diff
@@ -93,6 +93,11 @@ void EditorAssetLibraryItem::_notification(int p_what) {
 			category->add_theme_color_override("font_color", Color(0.5, 0.5, 0.5));
 			author->add_theme_color_override("font_color", Color(0.5, 0.5, 0.5));
 			price->add_theme_color_override("font_color", Color(0.5, 0.5, 0.5));
+
+			if (author->get_default_cursor_shape() == CURSOR_ARROW) {
+				author->add_theme_color_override("font_pressed_color", Color(0.5, 0.5, 0.5));
+				author->add_theme_color_override("font_hover_color", Color(0.5, 0.5, 0.5));
+			}
```

Comment:
Add a comment to explain why this is done:

```suggestion
			if (author->get_default_cursor_shape() == CURSOR_ARROW) {
				// Disable visible feedback if author link isn't clickable.
				author->add_theme_color_override("font_pressed_color", Color(0.5, 0.5, 0.5));
				author->add_theme_color_override("font_hover_color", Color(0.5, 0.5, 0.5));
			}
```